python/doc/examples/calibration/bayesian_calibration.ipynb

```diff
@@ -418,7 +418,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- Proposal distribution: uniform."
+    "Proposal distribution: uniform."
```

Comment:
```suggestion
    "Proposal distribution: Uniform."
```
I am not sure about this one. I think grammatically you are correct, but I think (I may be wrong) that we usually use uppercase letters for distributions in the doc. Or maybe it is simply in the context of the API.